lib/beaker-hostgenerator/data.rb

```diff
@@ -1125,6 +1125,28 @@ def osinfo
             'template' => 'win-2012r2-ja-x86_64'
           }
         },
+        'windows2012r2_fr-64' => {
+          :general => {
+            'platform'           => 'windows-2012r2-64',
+            'packaging_platform' => 'windows-2012-x64',
+            'ruby_arch' => 'x64'
+          },
+          :vmpooler => {
+            'template' => 'win-2012r2-fr-x86_64'
+            'user'     => 'Administrateur'
```

Comment:
Note that the Administrator Username also needs to be localised for this platform, otherwise ssh won't be able to connect to the host.